test/OpenTelemetry.Api.Tests/Trace/Propagation/TracestateUtilsTests.cs

```diff
@@ -14,9 +14,10 @@
 // limitations under the License.
 // </copyright>
 
+using OpenTelemetry.Context.Propagation;
 using Xunit;
 
-namespace OpenTelemetry.Context.Propagation.Tests
+namespace OpenTelemetry.Api.Tests.Trace.Propagation
```

Comment:
I think we should consider keeping the namespaces as is. It's pretty customary to prefix the namespaces of tests with the namespace containing the stuff being tested. So in this example, the stuff being tested is still contained in the `OpenTelemetry.Context.Propagation` namespace even though it's in the OpenTelemetry.Api package.

Reply:
Done.